pkg/setting/setting.go

```diff
@@ -1297,6 +1298,7 @@ func readServerSettings(iniFile *ini.File, cfg *Cfg) error {
 		return err
 	}
 
+	cfg.CDNPath = valueAsString(server, "cdn_path", "")
```

Comment:
@marefr think here we should probably add Grafana version string to the path (so v7.4.0-beta1) for example. (or in index.go) so that on the cdn each published release is under different paths , the config path you set in the ini file is just the root cdn path

Reply:
using a hash would work with master builds and pre-releases as well. That would be nice.

Reply:
So maybe both version and hash then? Or only using hash for pre-releases and version for stable ones, not sure how hard that would be.

Reply:
The asset files are already hashed, this should just add versio folder. Master builds already build a number of some sort I think

Reply:
True, makes sense